.github/workflows/test.yml

```diff
@@ -38,7 +38,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        gradleTask: [unitTest, connectedTest, azureTest]
```

Comment:
jsut for my understanding...what's the difference between `azureUnitTest` and `azureTest`?

Reply:
`azureTest` is set up for connected tests with Sam, RBS, etc.  But it is not set up for using the MockMvc.

Reply:
Maybe we should `s/azureTest/azureConnectedTest` for clarity?

So basically:
* unitTest -- non-Azure unit tests
* connectedTest -- non-Azure connected tests
* azureUnitTest -- Azure unit tests only
* azureConnectedTest -- Azure connected tests only